### PR TITLE
Adding the OpenIdConnectPromptParameter field UNSET

### PR DESCRIPTION
**Pull request by tanmaymanolkar1**
Fix for the github issue https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/1122

Added the field NOT_SENT and replaced the NONE, for the reasons of naming conventions

### CODE REVIEW

**Comment by shahzaibj** [comment]
Can you elaborate on the naming conventions?

**Comment by tanmaymanolkar1** [comment]
OpenIdConnectPromptParameter .NOT_SENT is used specifically to indicate that no prompt parameter would be injected into the Request. The OpenIdConnectPromptParameter.NONE naming was a bit ambiguous and this condition wasn't specifically being used initially, since there is a need to use this OIDC prompt parameter now, it would be better to correct the naming to resolve the ambiguity.

File: common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdConnectPromptParameter.java [hunk 1]
```diff
@@ -35,6 +35,11 @@
     /**
      * No prompt parameter will be injected into the request.
      */
+    NOT_SENT,
```
Comment (kreedula) [inline common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdConnectPromptParameter.java:38]: `NOT_SENT` seems a bit strange, how about `NOT_SET` , `NULL` . Shane suggested `UNSET` when we were discussing this as well

Comment (tanmaymanolkar1) [reply common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdConnectPromptParameter.java:38]: @kreedula sure UNSET would be good!

**Comment by iambmelt** [comment]
There is a document we need to update associated with these changes:
https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/docs/prompt_behavior.md

**Pull request merged**